docs/team/candyhy.md

```diff
@@ -9,3 +9,61 @@ PocketEstate enables easy organisation of mass clientele property information th
 location and housing type, that may otherwise be difficult to manage.
 
 Given below are my contributions to the project.
+
+**Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s2.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=&tabOpen=true&tabType=authorship&zFR=false&tabAuthor=candyhy&tabRepo=AY2021S2-CS2103T-T13-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
+
+#### New Features & Enhancements
+
+* **Enhancement**: Graphical User Interface (GUI)
+    * What it does: Simultaneously displays both property and appointment lists side by side
+    * Justification: This feature improves the product significantly because it enables integration of commands on both property and appointment lists, yet enables the user to conduct distinct commands on both lists and view its effects simultaneously without having to call `list property` or `list appointment` prior to each operation
+    * Highlights: Greying out of properties and appointments with expired deadlines in the GUI, a newly designed Light theme colour scheme with a organised, minimalist design that highlights important details.
+    The implementation was challenging due to limited experience and precise nature of fxml.
+
+<div style="page-break-after: always;"></div>
+
+* **Enhancement**: Maintains unique `Property` and `Appointment` lists, and all their attributes.
+    * What it does: Supports other commands like `add`, `edit`, `sort`, `filter` as all property and appointment data are maintained in a list. The unique property and appointment lists prevents addition/ edit of `Property` / `Appointment` that results in duplicate  `Property` / `Appointment` listings.
+    * Justification: The core functionality of our application is heavily reliant on both of these lists, as `Property` and `Appointment` lists are necessary to maintain `Property` and `Appointment` data PocketEstate manages.
+    * Highlights: Prevents addition of duplicate `Property` and `Appointment` to minimise redundant data.
+
+<div style="page-break-after: always;"></div>
+
+* **Enhancement**: Implemented storage support for both `Property`, `Appointment` and their attributes.
+    * What it does: Performs accurate parsing of `Property` and `Appointment` data from the respective `Property` and `Appointment` lists to data form that is reader friendly for display to the user, which is then stored in separate json files.
```

Comment:
would "lists to a data form that is reader friendly" be better?

Reply:
will correct my grammar!